java-support/src/main/scala/io/cloudstate/javasupport/impl/crdt/CrdtHelper.scala

```diff
@@ -0,0 +1,148 @@
+package io.cloudstate.javasupport.impl.crdt
+
+import java.lang.reflect.{Constructor, Executable}
+import java.util.function.Consumer
+import java.util.{function, Optional}
+
+import com.google.protobuf.{Any => JavaPbAny}
+import io.cloudstate.javasupport.crdt._
+import io.cloudstate.javasupport.impl.ReflectionHelper.{
+  InvocationContext,
+  MainArgumentParameterHandler,
+  MethodParameter,
+  ParameterHandler
+}
+import io.cloudstate.javasupport.impl.{ReflectionHelper, ResolvedType}
+import io.cloudstate.javasupport.{ServiceCall, ServiceCallFactory}
+
+import scala.reflect.ClassTag
+
+class CrdtHelper {
```

Comment:
Should this be a class? Seems like it should be an object?

Reply:
ok fixed